### PR TITLE
CTCTRADERS-834: Amend 'page-not-found' content and add link to NCTS

### PR DESCRIPTION
**Pull request by kenneth-rayner**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -39,6 +39,13 @@ phaseBanner.1 = This is a new service – your
 phaseBanner.2 = feedback
 phaseBanner.3 = will help us to improve it.
 
+pageNotFound.heading = Page not found
+pageNotFound.paragraph1 = If you typed the web address, check it is correct.
+pageNotFound.paragraph2 = If you pasted the web address, check you copied the entire address
+pageNotFound.paragraph3Start = If the web address is correct or you selected a link or button,
+pageNotFound.contactLink=contact the New Computerised Transit System(NCTS) helpdesk
```
Comment (PSouthern) [inline conf/messages.en:46]: Should these two lines not be combined as per the ticket requirements?
Also, I think the (opens in new window) part is required for accessibility standards.

File: conf/views/notFound.njk [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+
+{% block mainContent %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-xl">{{ messages("pageNotFound.heading") }}</h1>
+      <p class="govuk-body">{{ messages("pageNotFound.paragraph1") }}</p>
+      <p class="govuk-body">{{ messages("pageNotFound.paragraph2") }}</p>
+      <p class="govuk-body">{{ messages("pageNotFound.paragraph3Start") }}
+      <a  href={{"https://www.gov.uk/new-computerised-transit-system"}} class="govuk-link">{{messages("pageNotFound.contactLink") }}</a>
```
Comment (PSouthern) [inline conf/views/notFound.njk:18]: May require updating following resolution of above comments.

**Review by PSouthern — CHANGES_REQUESTED** [review]

File: conf/messages.en (hunk 1 above)
Comment (kenneth-rayner) [reply conf/messages.en:46]: Added the '(open in a new tab)' part. Having looked at the other services I have updated the link so it shows the full text as per ticket.

File: conf/views/notFound.njk (hunk 2 above)
Comment (kenneth-rayner) [reply conf/views/notFound.njk:18]: No change, still works fine

**Review by PSouthern — APPROVED** [review]

**Pull request merged**